src/utils/getFilteredTodos.ts

```diff
@@ -0,0 +1,21 @@
+import { FilterStatus } from '../types/FilterStatus';
+import { Todo } from '../types/Todo';
+
+export const getFilteredTodos = (todos: Todo[], filterStatus: FilterStatus) => {
+  if (filterStatus !== FilterStatus.ALL) {
+    return todos.filter(todo => {
+      switch (filterStatus) {
+        case FilterStatus.ACTIVE:
+          return !todo.completed;
+
+        case FilterStatus.COMPLETED:
+          return todo.completed;
+
+        default:
+          return true;
+      }
+    });
```

Comment:
I think we should avoid conditional nesting.  You can just use switch case with default value : return todos
```suggestion
     switch (filter) {
    case Filter.ACTIVE:
      return todos.filter(todo => !todo.completed);

    case Filter.COMPLETED:
      return todos.filter(todo => todo.completed);

    default:
      return todos;
```